app/routes/admin/email/components/RestrictedMailEditor.tsx

```diff
@@ -135,13 +135,13 @@ const RestrictedMailEditor = ({
         label="Møter"
         name="meetings"
         isMulti
-        placeholder="Møter du ønsker å sende epost til"
+        placeholder="Møter du ønsker å sende e-post til"
         filter={['meetings.meeting']}
         component={SelectInput.AutocompleteField}
       />
       <Field
         disabled={restrictedMailId}
-        label="Epost addresser"
+        label="E-post addresser"
```

Comment:
```suggestion
        label="E-post-adresser"
```
At least remove a `d` from addresser.